medusa/post_processor.py

```diff
@@ -1009,15 +1009,14 @@ def process(self):
                 self._log(u'File exists and the new file has the same size, aborting post-processing')
                 return True
 
-            if all([new_ep_quality <= old_ep_quality,
+            if all([new_ep_quality <= old_ep_quality, # Why we check if this ?!
```

Comment:
@pymedusa/developers any one knows why we check if `new_ep_quality <= old_ep_quality` ?